dora/core/common/src/main/java/alluxio/wire/WorkerInfo.java

```diff
@@ -33,7 +33,7 @@ public final class WorkerInfo implements Serializable {
   private WorkerIdentity mIdentity;
   private WorkerNetAddress mAddress = new WorkerNetAddress();
   private int mLastContactSec;
-  private String mState = "";
+  private WorkerState mState = WorkerState.UNRECOGNIZED;
```

Comment:
I had to assign a default value for worker state. A `UNRECOGNIZED` state means either the code that creates the `WorkerInfo` object did not set its state explicitly, or the worker is unknown to the membership service (e.g. never registered).